### DISCUSSION TITLE
How to provide default options in a multi select?

### DISCUSSION THREAD

**Discussion by joshembling** [filamentphp/filament#8289] [General]
Just looking at the example provided in the docs

```
Select::make('technologies')
    ->multiple()
    ->options([
        'tailwind' => 'Tailwind CSS',
        'alpine' => 'Alpine.js',
        'laravel' => 'Laravel',
        'livewire' => 'Laravel Livewire',
    ])
```

If I chain something like this: `->default('tailwind')`
Or this `->default(['tailwind'])`

Neither seem to have any effect and no default options are selected. 

Anyone have any ideas?

**Comment by danharrin** [comment]
`->default(['tailwind'])` is correct, are you using the admin panel

**Comment by zawilliams** [comment]
Running into this right now and it seems to not work. @danharrin, I've tried `->default(['tailwind'])` as you said it's the correct usage but I'm not seeing defaults show when `->multiple()` is added.

```php
Select::make('technologies')
    ->multiple()
    ->options([
        'tailwind' => 'Tailwind CSS',
        'alpine' => 'Alpine.js',
        'laravel' => 'Laravel',
        'livewire' => 'Laravel Livewire',
    ])
    ->default(['laravel', 'livewire']),
```

The select is just empty.

**Reply by danharrin** [reply]
Remember that defaults only apply on the "create" form and not "edit"

**Reply by zawilliams** [reply]
Ah, got it - got it working. Thanks @danharrin!
